contracts/cw-proposal-multiple/src/proposal.rs

```diff
@@ -35,6 +35,10 @@ pub struct MultipleChoiceProposal {
     /// Information about the deposit that was sent as part of this
     /// proposal. None if no deposit.
     pub deposit_info: Option<CheckedDepositInfo>,
+    /// The timestamp at which this proposal was created.
+    pub created: Timestamp,
+    /// The timestamp at which this proposal's status last changed (e.g. Passed, Executed).
```

Comment:
Could we add a note here about how quorum expiration stuff makes this a little strange? Technically, it's possible for a proposal to become "passed" without this being updated so we don't want folks relying on this to be up to date.

Reply:
Hmm so looking at this more it looks like we currently don't update this when passed anywhere. Is that intentional? If not, should we be updating this in `execute_vote` if it causes the proposal to pass?

Reply:
so the timestamp is updated in `update_status` which is called by `execute_vote,` I think that should suffice?

Reply:
Oh I see. Yeah that's great!